src/documents/ContextAwareness.md

```diff
@@ -0,0 +1,327 @@
+            <td>One or more <code>gradeNames</code> to be the result if none of the <code>checks</code> entries matches</td>
+        </tr>
+        <tr>
+            <td><code>priority</code></td>
+            <td><code>Priority</code> value - see [Priorities](Priorities.md) for a full explanation</td>
+            <td>The priority (if any) that the <code>gradeNames</code> resulting from this dimension should have over those resulting from any other dimension. This should be an entry of the form
+            <code>before:adaptationName</code> or <code>after:adaptationName</code> for one of the other dimensions attached to this component within <code>contextAwareness</code></td>
+        </tr>
+    </tbody>
+</table>
+
+The result of the `contextAwareness` record is that a number of the elements within `checks` will be evaluated in the visible context, and result in a number of `gradeNames` which will then be
+contributed into the `gradeNames` of the instantiating component in a particular order. This order is governed by both the `priority` entry at the adaptation level as well as at the check level.
+
+##Structure of members in a `checkRecord`
+
+The `checkRecord` structure which is used in the first row of this table is described now:
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="3">Members of a <code>checkRecord</code> entry within a <code>adaptationRecord</code> block of a <code>fluid.contextAware</code> component</th>
+        </tr>
+        <tr>
+            <th>Member</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>contextValue</code></td>
+            <td><code>String</code> <a href="IoCReferences.md">IoC reference</a></td>
+            <td>A standard IoC reference, either consisting of a bare context reference such as <code>{contextName}</code> or to a precise value such as <code>{contextName}.further.path</code> which should be sought by this check.
+            If a bare context name is supplied, the checked path will default to <code>options.value</code>. If the context name does not match, the expression will evaluate to <code>undefined</code></td>
+        </tr>
+        <tr>
+            <td><code>equals</code> (optional)</td>
+            <td><code>String</code>, <code>Number</code> or <code>Boolean</code></td>
+            <td>A value with which the context value fetched from <code>contextValue</code> should be checked. If omitted, this defaults to <code>true</code></td>
+        </tr>
+        <tr>
+            <td><code>gradeNames</code> (optional)</td>
+            <td><code>String</code> or <code>Array of String</code></td>
+            <td>One or more <code>gradeNames</code> that will be returned out to the <code>contextAwareness</code> system if this check passes. The check will pass if the context value referenced in <code>contextValue</code> can be found, and
```

Comment:
The conditions for a check passing should probably not be buried in the description of an optional member that isn't part of the determination. This might be better places in the 'equals' description.